src/krux/printers/thermal.py

```diff
@@ -201,7 +201,7 @@ def has_paper(self):
         # Bit 2 of response seems to be paper status
         res = self.uart_conn.read(1)
         if res is None:
-            return False
+            return None
```

Comment:
?

Reply:
I was getting value error raise when printer was not connected. I will do a better fix for this.
By the way, I printed with Amigo, made some small changes later on this topic too.
[printing](https://photos.app.goo.gl/7SyEPrKSU4z929Lw9)
[scanning](https://photos.app.goo.gl/SSzkPEJR2JwpZfbi9)